lib/ansible/modules/cloud/cloudstack/cs_instance_facts.py

```diff
@@ -184,6 +185,8 @@
 
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.cloudstack import AnsibleCloudStack, cs_argument_spec
+from collections import defaultdict
```

Comment:
Please put these *stdlib* imports above project imports separating sections with an empty line.

Reply:
The convention is to group imports:
1) stdlib imports
2) common third-party imports
3) main framework imports (if applicable, not in this case)
4) own project imports (ansible, in our case)